### ISSUE TITLE
Libguides: Role with implied hidden content has keyboard focus

### ISSUE THREAD

**Issue by rschwab** [UCSCLibrary/SiteImprove#53]
Some elements have a role that implies their internal structure should be hidden from assistive technologies. In other words, their sub-elements have a presentational role.

If a sub-element has keyboard focus, it should be moved outside of the presentation role.

[Siteimprove link](https://my2.siteimprove.com/Accessibility/624596/77184128804/NextGen/Issue/1?conformance=&siteTargetIssueKinds=1,2&wcagVersion=21&pageSegments=&ruleName=sia-r90&ruleId=90&issueKind=1&tagFilterType=2&siteTarget.wcagVersion=21&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2)

[csv](https://github.com/user-attachments/files/18713877/SC.-.Library-Pages.with.a.specific.issue-Table.7.csv)

**Comment by rschwab** [comment]
I want to get better action item on this before I consider it ready

**Comment by JessWaggs** [comment]
From what I see on 08/05/2026, all remaining occurrences are due to LibGuides side navigation. This could be a problem Springshare needs to solve OR it could be due to our creative use of the tab-nav template (wherein we have tabs Springshare created to display horizontally instead display vertically as a side-nav).
